sdk/formrecognizer/ai-form-recognizer/src/generated/lro/azureAsyncPolling.ts

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
these lro files should be deleted.

Reply:
Is there any way to tell autorest not to generate them?

Reply:
Latest version does not generate them.